src/main/java/Manager.java

```diff
@@ -0,0 +1,155 @@
+                + "\t  " + todo.toString();
+        String sizeMessage = "\tNow you have " + listSize + " tasks in the list.";
+
+        String[] messagePacket = { addedMessage, sizeMessage };
+        Io.printMessage(messagePacket);
+    }
+
+    public static void addEvent(String inputMessage, String from, String to) {
+        Task event = new Event(inputMessage, from, to);
+        if (listSize >= 100) {
+            System.out.println("List size exceeded!");
+            return;
+        }
+        taskList[listSize] = event;
+        listSize++;
+
+        String addedMessage = "\tGot it. I've added this task:\n"
+                + "\t  " + event.toString();
+        String sizeMessage = "\tNow you have " + listSize + " tasks in the list.";
+
+        String[] messagePacket = { addedMessage, sizeMessage };
+        Io.printMessage(messagePacket);
+    }
+
+    public static void markTask(int taskNum) {
+        Task currentTask = taskList[taskNum - 1];
+        if (taskNum > listSize || taskNum <= 0) {
+            System.out.println("\tNo such task exists! Please try again");
+            return;
+        }
+        currentTask.markDone();
+
+        String markMessage = "\tNice! I've marked this task as done:\n"
+                + "\t  " + currentTask.toString();
+
+        String[] messagePacket = { markMessage };
+        Io.printMessage(messagePacket);
+    }
+
+    public static void unmarkTask(int taskNum) {
+        Task currentTask = taskList[taskNum - 1];
+        if (taskNum > listSize || taskNum <= 0) {
+            System.out.println("\tNo such task exists! Please try again");
+            return;
+        }
+        currentTask.markUndone();
+
+        String unmarkMessage = "\tOK, I've marked this task as not done yet:\n"
+                + "\t  " + currentTask.toString();
+
+        String[] messagePacket = { unmarkMessage };
+        Io.printMessage(messagePacket);
+    }
+
+    public static void printTasks() {
+        String[] messagePacket = new String[listSize + 1];
+        messagePacket[0] = "\tHere are the tasks in your list:";
+        int messageCount = 1;
+
+        for (int i = 0; i < listSize; i++) {
+            String line = "\t" + (i + 1) + ". " + taskList[i];
+            messagePacket[messageCount++] = line;
+        }
+        Io.printMessage(messagePacket);
+    }
+
+    public static void handleCommand(String[] inputArray) {
+        String command = inputArray[0];
+
+        switch (command) {
+        case "todo":
+            String todoDetails = inputArray[1];
+            addTodo(todoDetails);
+            break;
+
+        case "deadline":
+            if (!inputArray[1].contains("/by")) {
+                System.out.println("\tWrong format! Please include /by.");
+                break;
+            }
+
+            String[] deadlineDetails = inputArray[1].split(" /by ", 2);
+            String deadlineDescription = deadlineDetails[0];
+            String by = deadlineDetails[1];
```

Comment:
Perhaps these lines of code can be implemented in the addDeadline() method to improve readability. Similarly for the case "event" below.